container/options.go

```diff
@@ -216,6 +218,19 @@ func SplitPercent(p int) SplitOption {
 	})
 }
 
+// SplitFixed sets the heights of the top and the bottom containers.
```

Comment:
We should probably update this comment so that it no longer references "top" or "bottom". The way we have it now, it works on either horizontal or vertical splits.

Maybe we can borrow some language from the comment above SplitPercent?